src/Classes/Utils/Presets.ts

```diff
@@ -0,0 +1,206 @@
+
+export class Presets {
+    public static getPreset(key: PresetType): RecursivePartial<IOptions> | undefined {
+        return this.available[key.toString()];
+    }
+
+    private static get available(): { [key: string]: RecursivePartial<IOptions> } {
+        return {
+            [PresetType.basic]: this.basic,
+            [PresetType.backgroundMask]: this.backgroundMask,
+            [PresetType.fontAwesome]: this.fontAwesome,
+            [PresetType.snow]: this.snow,
+            [PresetType.bouncing]: this.bouncing,
+            [PresetType.stars]: this.stars,
+        }
+    }
+
+    private static get basic(): RecursivePartial<IOptions> {
+        return {
+            particles: {
+                color: {
+                    value: "#ffffff",
+                },
+                lineLinked: {
+                    color: "#ffffff",
+                    distance: 150,
+                    enable: true,
+                    opacity: 0.4,
+                    width: 1,
+                },
+                move: {
+                    direction: MoveDirection.none,
+                    enable: true,
+                    outMode: OutMode.out,
+                    speed: 2,
+                },
+            },
+        };
+    }
+
+    private static get backgroundMask(): RecursivePartial<IOptions> {
+        return {
+            particles: {
+                color: {
+                    value: "#ffffff",
+                },
+                shape: {
+                    type: ShapeType.circle,
+                },
+                opacity: {
+                    value: 1,
+                    random: false,
+                },
+                size: {
+                    value: 30,
+                    random: true,
+                },
+                lineLinked: {
+                    enable: true,
+                    distance: 150,
+                    color: "#ffffff",
+                    opacity: 1,
+                    width: 1,
+                },
+                move: {
+                    enable: true,
+                    speed: 2,
+                    direction: MoveDirection.none,
+                    random: false,
+                    straight: false,
+                    outMode: OutMode.out,
+                }
+            },
+            interactivity: {
+                events: {
+                    onHover: {
+                        enable: true,
+                        mode: HoverMode.bubble,
+                    },
+                },
+                modes: {
+                    bubble: {
+                        distance: 400,
+                        size: 100,
+                        duration: 2,
+                        opacity: 1,
+                    },
+                },
+            },
+            backgroundMask: {
+                enable: true,
+                cover: {
+                    value: {
+                        r: 255,
+                        g: 255,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'g' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)